xrpl/asyncio/clients/json_rpc_base.py

```diff
@@ -3,6 +3,7 @@
 from __future__ import annotations
```

Comment:
You may want to update the client base type and cover websockets too (cc @desimmons)

Please also update the changelog